### ISSUE TITLE
Label needed on "What's new?" textarea on News Feed

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#375]
Area of site: News Feed
URL: https://hcommons-staging.org/activity/

**Description**

The textarea for the "What's new, ___?" form lacks a label. 

**Recommendations**

Refactor the markup of the label from a `<p>` to a `<label>` and add a `for="whats-new"` attribute to it
